### DISCUSSION TITLE
Improvement for parsing comments to descriptions

### DISCUSSION THREAD

**Discussion by zhenwenc** [MichalLytek/typegraphql-prisma#185] [Ideas]
Thanks a lot for creating and maintaining this tool, I found it super useful! 🙏 

There is a potential issue I encountered recently around the fields comments:

#### Multi-line comments should be concatenated with a space
```
// schema.prisma
model MyTable {
  /// Line1
  /// Line2
  id String @id
}
```

```typescript
// in generated model
TypeGraphQL.Field(_type => String, {
    nullable: true,
    description: "Line1Line2"  // Expected: "Line1 Line2"
})
```

#### Should replace multiple consecutive empty lines with `\n` (simular behavior in markdown)
```
// schema.prisma
model MyTable {
  /// Line1
  ///
  /// Line2
  id String @id
}
```

```typescript
// in generated model
TypeGraphQL.Field(_type => String, {
    nullable: true,
    description: "Line1 Line2"  // Expected: "Line1\nLine2"
})
```

#### Invalid code will be generated when the comment contains newline character `\n`

**These are the current behavior: (left: prisma schema, right: the generated code)**
- `/// blah "\n" blah` => `description: "blah \"\\", blah, ": ` (cause the whole file to be invalid)
- `/// blah "\r\n" blah` => `description: "blah \"\\\" blah"`
- `/// blah "\\n" blah` => `description: "blah \"\\\" blah"`

---

Created this discussion as I am not sure if these are expected behavior or bugs, and some of them are suggested improvements. Thanks in advance for the help! 🙏

**Comment by MichalLytek** [comment]
Why do you think it should not maintain the original line breaks?

**Reply by zhenwenc** [reply]
@MichalLytek Thanks for your reply. Sorry if I didn't elaborate on it clearly. I do expect the library to maintain the line breaks, but it doesn't from my inspection.

**Comment by MichalLytek** [comment]
Thanks for the report. It's fixed by cf1f0c1 🔒

**Reply by zhenwenc** [reply]
Thank you very much for the fix! 🌟
